contrib/linearize/linearize-hashes.py

```diff
@@ -96,7 +96,7 @@ def get_block_hashes(settings, max_blocks_per_call=10000):
 
 def get_rpc_cookie():
     # Open the cookie file
-    with open(os.path.join(os.path.expanduser(settings['datadir']), '.cookie'), 'r') as f:
+    with open(os.path.join(os.path.expanduser(settings['datadir']), '.cookie'), 'r', encoding="utf8") as f:
```

Comment:
The cookie file is ASCII encoded

Reply:
I'm not sure that's a problem: if something is ASCII-encoded, it doesn't hurt reading it as utf-8.

Reply:
No strong opinion. Would throw and error if it was no longer ascii.